ui/app/controllers/vault/cluster/settings/mount-secret-backend.js

```diff
@@ -22,8 +22,5 @@ export default Controller.extend({
         this.wizard.transitionFeatureMachine(this.wizard.featureState, 'CONTINUE', type);
       });
     },
-    onConfigError: function(modelId) {
```

Comment:
this was not being used, and we don't actually want to transition to an error page. We want to keep the user on the page because it could just be they don't have access to create that secret engine with that name.